src/handler/newOrder.ts

```diff
@@ -121,13 +122,15 @@ function getOrderAndFeeFactor(simpleOrder, rate, tokenList, tokenConfigs, config
   const order = {
     makerAddress: config.mmProxyContractAddress.toLowerCase(),
     makerAssetAmount,
+    makerAssetAddress: makerToken.contractAddress,
     makerAssetData: assetDataUtils.encodeERC20AssetData(
       getWethAddrIfIsEth(makerToken.contractAddress, config)
     ),
```

Comment:
keep line L125, and remove this encoding.

only do token address encoding in pmmv5. ammv1 don't need this either.

Reply:
Fixed in 5b29dea